src/stylesheets/core/_utilities.scss

```diff
@@ -16,6 +16,13 @@
   left: -999em;
 }
 
+// Aria hidden helper
+@mixin aria-hidden() {
```

Comment:
@msecret & @maya - I think having it as a mixin also establishes a pattern of how to handle `display: none` related specifically to accessibility considerations - similar to the `unstyled-list` mixin.

I don't think this particular pattern is used with any other UI component. I think the problem we were trying to solve initially was that `aria-hidden` shouldn't `display: none` on elements that use that attribute setting. @maya's simple solution solves this problem and limits its scope to the `accordion-content` class. This PR extends the flexibility of that implementation, which may be a pre-optimization (capability with no demand from developers, internal nor external)...??